overlord/snapstate/snapstate.go

```diff
@@ -314,6 +314,50 @@ func refreshRetain(st *state.State) int {
 	return retain
 }
 
+func MaxInhibitionTime(st *state.State) time.Duration {
+	// deduct 1s so it doesn't look confusing initially when two notifications
+	// get displayed in short period of time and it immediately goes from "14 days"
+	// to "13 days" left.
+	return time.Duration(maxInhibitionDays(st))*24*time.Hour - time.Second
+}
+
+// maxInhibitionDays returns refresh.max-inhibition-days value if set, or the default value (14 days).
+// It deals with potentially wrong type due to lax validation.
+func maxInhibitionDays(st *state.State) int {
+	var val interface{}
+	// due to lax validation of refresh.max-inhibition-days on set we might end up having a string representing a number here; handle it gracefully
+	// for backwards compatibility.
+	err := config.NewTransaction(st).Get("core", "refresh.max-inhibition-days", &val)
```

Comment:
I think this was true for refresh.retain on which this code is based, but since you are adding a new entry which already has proper validation you can use `int` here.

Reply:
Done.